src/optimizer/query_planner.c

```diff
@@ -11567,7 +11567,14 @@ qo_plan_scan_print_text (FILE * fp, QO_PLAN * plan, int indent)
   switch (plan->plan_un.scan.scan_method)
     {
     case QO_SCANMETHOD_SEQ_SCAN:
-      fprintf (fp, "TABLE SCAN (%s)", class_name);
+      if (plan->plan_un.scan.agg_optimized)
+	{
+	  fprintf (fp, "INDEX AGG OPTIMIZED (%s)", class_name);
+	}
+      else
+	{
+	  fprintf (fp, "TABLE SCAN (%s)", class_name);
+	}
```

Comment:
Since it is determined by the executor, it is not possible to output to the plan.

Reply:
However, the plan could be different from executing. Therefore, we need to print both of the plan and scan method  in executing together.

Reply:
I don't think so. Inaccurate plan information will only confuse users.